Apps/UnitTests/Scripts/tests.js

```diff
@@ -31,21 +31,21 @@ const expect = chai.expect;
 describe("XMLHTTPRequest", function () {
     this.timeout(0);
     it("should have readyState=4 when load ends", async function () {
-        const xhr = await makeXHR("GET", "https://babylonjs.com");
+        const xhr = await createRequest("GET", "https://babylonjs.com");
         expect(xhr.readyState).to.equal(4);
     })
     it("should have status=200 for a valid page", async function () {
-        const xhr = await makeXHR("GET", "https://babylonjs.com");
+        const xhr = await createRequest("GET", "https://babylonjs.com");
         expect(xhr.status).to.equal(200);
     })
     it("should have status=404 for a page that does not exist", async function () {
-        const xhr = await makeXHR("GET", "https://babylonjs.com/invalid");
+        const xhr = await createRequest("GET", "https://babylonjs.com/invalid");
         expect(xhr.status).to.equal(404);
     })
 
 });
 
 
-mocha.run(function (failures) {
+mocha.run( failures => {
```

Comment:
nit: there typically isn't a space here.  might consider auto formatting the whole file.
```suggestion
mocha.run(failures => {
```

Reply:
fixed! thanks! would it be possible to add a linting step to the CI checks?

Reply:
We could. We probably should transition this to a TS file before doing that though.

Reply:
What should we use to compile TS files? As far as I know we are not using TS anywhere in the Babylon Native repo yet

Reply:
I'm not sure yet. Something similar to how Babylon.js does it probably.